src/staticfile/finalize/data.go

```diff
@@ -80,15 +80,25 @@ http {
     root <%= ENV["APP_ROOT"] %>/public;
 
    {{if .ForceHTTPS}}
+	 set $updated_host $host;
+	 if ($http_x_forwarded_host != "") {
+       set $updated_host $http_x_forwarded_host;
+     } 
+
      if ($http_x_forwarded_proto != "https") {
-       return 301 https://$host$request_uri;
-     }
+	   return 301 https://$updated_host$request_uri;
```

Comment:
@dmarmugi This is minor, but would you mind fixing some of the formatting in this file?

Reply:
sure @ameyer-pivotal, i think the ide put in tabs. two options, though -- the `{{if .ForceHTTPS}}` block _was_ on a 3-space indent. I could move the whole block to fall in line with the rest of the file, or just make sure that my update falls in with the formatting that was there before.

Reply:
We just want it to be consistent, so either option is fine with us 😉